.pre-commit-config.yaml

```diff
@@ -46,6 +46,8 @@ repos:
         args:
           - --min-py3-version
           - "3.4"
+	  - max-py-version
+	  - "3.10"
```

Comment:
The indentation looks off - but I am not sure.

Reply:
Oops, my diff tool added tabs! 😱